Guides/organising.md

```diff
@@ -6,20 +6,26 @@ We've described how a typical meeting would run in the ['Facilitating'](./facili
 
 ## Deciding on material to read
 
-The [reading list](/READING-LIST.md) is organised into different sections - some are philosophical, some are technical. Our [previously discussed pieces are available in our Sessions Overview](/SESSIONS.md). We like to check off things we have discussed [in the reading list](/READING-LIST.md).
+The [reading list](/READING-LIST.md) is a curated list of what we, and others, have added over the years. It also shows what we have currently read, [more detail about previously discussed books is in our Sessions Overview](/SESSIONS.md).
 
-The whole group votes on what to read for the following meeting at the end of every session. A slido is pre-populated with items already on our reading list, and attendees are invited to vote for items, as well as add anything they want to read for others to upvote.
+From 2025, the organisers will each pick a book to read and decide which one should be part of a future session. In general, we decide based on the following:
+- Is the material engaging and interesting?
+- Have we covered the topic before?
+- Is the book affordable and available?
+- Is the information and material up-to-date?
 
 ## Administrative tasks
 
+More information on the tools we use can be found in our ['Tools'](./tools.md) page.
+
 ### Before the meeting
 
-- [ ] Determine the material to discuss from the item with most votes on the slido from the previous meeting
+- [ ] Bring suggestions to organiser meetings, divide material up between organisers, and decide which material should be discussed at the next event
 - [ ] Decide date of next meeting
 - [ ] Create a [meeting information file](./../Sessions/session-template.md) in the `Sessions/` folder
 - [ ] Update the [meeting overview file](./../SESSIONS.md) with a link to the new meeting information page
-- [ ] Create a sign-up form to manage attendees (we currently use the Ticket Tailor platform for this, but are open to other suggestions)
-- [ ] Produce a newsletter post promoting the event
+- [ ] Create a sign-up form to manage attendees (using Ticket Tailor)
```

Comment:
can we not list the tools here? can we link to our tools page instead? also below re newsletter

Reply:
and event- it just means fewer changes to make in future